t3xupdate.py

```diff
@@ -58,11 +70,12 @@ def get_version(self):
         return self.get_raw_version()[-4:]
 
     def get_product(self):
-        return self.get_raw_version().split(b'_')[2]
+        x = self.get_raw_version()
+        return x.split(b'_')[1]
 
     def parse_update(self, file):
         self.file = open(file, 'rb')
-        if self.file.read(4) == b'JCID':
+        if self.file.read(5) == b'AIXUN':
```

Comment:
I'm not the repo owner, but this will break updates for JCID products.  You probably were just trying to get it to work for T413.